example/25-rbac.yaml

```diff
@@ -50,3 +50,39 @@ rules:
     - update
     - get
     - watch
+---
```

Comment:
Can you add a comment for what we need this `ClusterRole` and where it's supposed to be applied (shoot)?